Detections/ASimNetworkSessions/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,40 @@
+id: D66DD819-F2E6-43F7-8BC1-70BDE8011B39
+name: Several deny actions registered (AsimWebSession)
+description: |
+  'Identifies attack pattern when attacker tries to move, or scan, from resource to resource on the network and creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
+tags:
+  - Id: f8dad4e9-3f19-4d70-ab7f-8f19ccd43a3e
+    version: 1.0.0
+  - Schema: ASIMNetworkSessions
```

Comment:
We are at 0.2.0

Reply:
Assuming you mean the Schema version, not the Parent detection version".

Reply:
Yup

Reply:
It will be 0.2.1 in a few hours by the way...